admin.php

```diff
@@ -42,10 +42,10 @@ function sqlQuery($statement, $link)
 <html>
 <head>
 <title>OpenEMR Site Administration</title>
-<link rel="stylesheet" href="public/assets/bootstrap/dist/css/bootstrap.min.css" type="text/css">
+<link rel="stylesheet" href="public/assets/bootstrap/dist/css/bootstrap.min.css">
 <script type="text/javascript" src="public/assets/jquery/dist/jquery.min.js"></script>
 <script type="text/javascript" src="public/assets/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
-<link rel="stylesheet" href="public/assets/@fortawesome/fontawesome-free/css/all.min.css" type="text/css">
+<link rel="stylesheet" href="public/assets/@fortawesome/fontawesome-free/css/all.min.css">
 <link rel="shortcut icon" href="public/images/favicon.ico" />
 <style>
     .oe-pull-away{
```

Comment:
Also you might wanna consolidate the internal stylesheet here but you don't have to do this in this pull

Reply:
OK, I will refine it in new PR, Thanks for your mention.